frontend/src/components/contexts/contexts.ts

```diff
@@ -1,14 +1,22 @@
 import * as React from "react";
-import { MockProfileBackendService } from 
-  "../../services/mock_profile_backend/mock_profile_backend";
-import { ProfileHandlerService } from 
+import { MockProfileBackendService } from
+  "../../services/mock_backend/mock_profile_backend";
+import { MockYourClubsBackendService } from
+  "../../services/mock_backend/mock_your_clubs_backend";
+import { MAX_NUM_CLUBS } from
+  "../../utils/temporary_testing_consts";
+import { NUM_PROFILES } from
+  "../../utils/temporary_testing_consts";
+import { ProfileHandlerService } from
   "../../services/profile_handler_service/profile_handler_service";
-
-const NUM_PROFILES = 10;
+import { YourClubsHandlerService } from
+  "../../services/your_clubs_handler_service/your_clubs_handler_service";
 
 /** TODO: Change to real backend service implementation */
```

Comment:
```suggestion
/** TODO: Change to real backend service implementation. */
```
Period at end of comment.